Core/Utilities/File.py

```diff
@@ -96,17 +121,17 @@ def getSize( fileName ):
   """Get size of a file.
 
   :param string fileName: name of file to be checked
-  
-  The os module claims only OSError can be thrown, 
+
+  The os module claims only OSError can be thrown,
   but just for curiosity it's catching all possible exceptions.
 
-  .. warning:: 
+  .. warning::
      On any exception it returns -1.
-  
+
   """
   try:
     return os.stat( fileName )[6]
-  except Exception:
+  except OSError:
     return - 1
```

Comment:
I didn't change this one, it was here before. I also find it odd. Should I change it? If I do, other files will have consequences.